README.md

```diff
@@ -143,8 +159,16 @@ In this example, we set a default for `optionsFile`, which is used to load a sam
 }
 ```
 
-Here are some examples of the output that results from using the above with various combinations of command line
-parameters and environment variables on a UNIX-like system:
+Note the `gradeNames` parameter refers to the `my.launcher.worker` grade we defined in the same file where we defined
+our launcher grade itself.  The `gradeNames` keyword in a configuration file is the the primary way in which we
+associate our launcher with the component grade(s) to be launched.
+
+As with sub-components, we can have multiple `gradeNames`, which will be merged as they would when instantiating a
+sub-component.  [Grades are merged from left to right](http://docs.fluidproject.org/infusion/development/ComponentGrades.html#combining-grades),
+so that that rightmost grade's options take precedence.
+
+So, assuming the above launcher and configuration file, here are some examples of the output that results when using 
+various combinations of command line parameters and environment variables:
```

Comment:
This change is very helpful, and very much addresses the issues I raised in [GPII-2457](https://issues.gpii.net/browse/GPII-2457). Thanks, @the-t-in-rtf!